python/docs/source/user_guide/sql/python_data_source.rst

```diff
@@ -137,3 +271,21 @@ Use the fake datasource with a different number of rows:
     # |  Caitlin Reed|1983-06-22|  89813|Pennsylvania|
     # | Douglas James|2007-01-18|  46226|     Alabama|
     # +--------------+----------+-------+------------+
+
+Start a streaming query with the fake data stream. Once we register the python data source, we can use it as source of readStream() or sink of writeStream() by passing short name or full name to format().
+
+.. code-block:: python
+
+    query = spark.readStream.format("fake").load().writeStream().format("fake").start()
```

Comment:
shall we add an example for writeStream?

Reply:
Also, do we have any example outputs we can show to users?

Reply:
Yeah, the output from console sink would be OK. Maybe batch 0 or so.

Reply:
Example output added.